test/bats/conjur.bats

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bats
+
+load helpers
+
+BATS_TESTS_DIR=test/bats/tests/conjur
+WAIT_TIME=180
+SLEEP_TIME=1
+
+CONJUR_NAMESPACE=conjur
+CONJUR_DATA_KEY="$(openssl rand -base64 32)"
+CONJUR_ACCOUNT=default
+CONJUR_URL=conjur-conjur-oss.conjur.svc.cluster.local
+
+EXPECTED_USERNAME="some_user"
+EXPECTED_PASSWORD="SecretPassword1234!"
+
+@test "allow conjur provider token requests and enable rotation" {
+  helm upgrade csi-secrets-store manifest_staging/charts/secrets-store-csi-driver --namespace=kube-system \
+    --set tokenRequests[0].audience="conjur" \
+		--set enableSecretRotation=true \
+		--set rotationPollInterval=30s
+  
+  kubectl wait --for=condition=ready --timeout=${WAIT_TIME}s --namespace=kube-system pod -l "app=secrets-store-csi-driver"
+}
```

Comment:
Could you add `--set tokenRequests[2].audience="conjur"` to https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/5f6daa036572b1cda059c20ecefb2ad4a642e6f8/Makefile#L439-L440 instead and remove this step?